### PR TITLE
Add cn-north-1 and cn-northwest-1

### PR DESCRIPTION
**Pull request by Kern--**
### Related Issue
N/A

### Proposed Changes
Adds support for cn-north-1 and cn-northwest-1

### Reviewers
@turan18

### CODE REVIEW

**Review by turan18 — APPROVED** [review]
LGTM

File: functions/source/soci-index-generator-lambda/Makefile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 default:
-	GOOS=linux GOARCH=amd64 go build -o main
+	# Use static builds to make sure we don't have library version issues between the build env and lambda
+	GOOS=linux GOARCH=amd64 go build -tags "osusergo netgo static_build" -ldflags '-extldflags "-static"' -o main
```
Comment (sparr) [inline functions/source/soci-index-generator-lambda/Makefile:3]: This seems like a bigger change than just adding region support

Comment (Kern--) [reply functions/source/soci-index-generator-lambda/Makefile:3]: I was having glibc version issues when testing in China. I think its useful to tie this change to the additional regions

**Pull request merged**